dspace-api/src/main/java/org/dspace/content/dao/impl/CommunityDAOImpl.java

```diff
@@ -75,7 +76,10 @@ protected CommunityDAOImpl() {
             query.setMaxResults(limit);
         }
         query.setParameter(sortField.toString(), sortField.getID());
-        return list(query);
+
+        // Convert List to HashSet and back again to filter out duplicates (e.g. a community
+        // having multiple titles/translations), without changing the order of elements
+        return new LinkedList<Community>(new LinkedHashSet<>(list(query)));
```

Comment:
I guess that the page size is not respected in this way. If we have 11 communities and one of that has title in multiple language result in multiple entry in the first page, flatting this list will probably left less than 10 result in the first page

Reply:
Thanks @abollini for catching that.
You're right, the limit happens prior to excluding the duplicates, which implies there can be less than 10 results.

What would you suggest is the best approach to avoid this? Would it make sense to only retrieve dc.title with place 0?

We can add an IT for this setup to verify when it is fixed

Reply:
Good point @abollini , I completely overlooked that.

@benbosman : Is there any way for us to filter these duplicates out via the database query (i.e. SQL/Hibernate)?  It seems like duplicate entries should come back with the same UUID...can we then use either a `DISTINCT` or `GROUP BY` clause to filter out duplicates?  Or am I misunderstanding the core issue here?